src/core/dev_api/src/itt.hpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Defines openvino domains for tracing
+ * @file itt.hpp
+ */
+
+#pragma once
+
+#include <openvino/cc/factory.h>
+#include <openvino/cc/selective_build.h>
+
+#include <openvino/itt.hpp>
+
+namespace ov {
+namespace itt {
```

Comment:
@ilya-lavrenov I've created a new cmake target (`ov_core_dev_api`) as a place for internal ops (a concept of internal ops was discussed with @jane-intel ). Do you have some suggestions to avoid copying `itt.hpp` header from core? (some common place?)

Reply:
please, see my another suggestion about putting operations to https://github.com/openvinotoolkit/openvino/tree/master/src/common/transformations/include/ngraph_ops

Reply:
done